string human_footprint_frame_id
geometry_msgs/PointStamped target
float64 distance_to_robot # distance between human & robot (m)
float64 alpha # desired angle between robot and human
---
bool success
geometry_msgs/PoseStamped result_pose